packages/runtime/src/widgets/Form/Dropdown.tsx

```diff
@@ -302,6 +303,7 @@ const Dropdown: React.FC<DropdownProps> = (props) => {
       <div ref={rootRef} style={{ flex: 1, ...formItemStyles }}>
         <EnsembleFormItem values={values}>
           <Select
+            allowClear={values?.allowClear || false}
```

Comment:
Nit: could just use double bang !!

Reply:
@evshi eslint doesn't allow me to do that. It suggests using Boolean as its alternative

Reply:
Then listen to eslint :)

Reply:
@evshi had to change it to use `??` to make it true by default